spec/lib/hash_sorter_spec.rb

```diff
@@ -1,37 +0,0 @@
-require "rails_helper"
-
-describe HashSorter do
-  describe ".sort" do
-    let(:links) do
-      {
-        group_2: [
-          { base_path: "/group-1/link-1", api_path: "/api/content/group-1/link-1" },
-          { base_path: "/group-1/link-2", api_path: "/api/content/group-1/link-2" },
-          { base_path: "/group-1/link-3", api_path: "/api/content/group-1/link-3" },
-        ],
-        group_1: [
-          { base_path: "/group-2/link-3", api_path: "/api/content/group-2/link-3" },
-          { base_path: "/group-2/link-2", api_path: "/api/content/group-2/link-2" },
-          { base_path: "/group-2/link-1", api_path: "/api/content/group-2/link-1" },
-        ],
-      }
-    end
-    let(:result) { described_class.sort(links) }
-
-    it "sorts the top-level keys" do
-      expect(result.keys).to eq(%i[group_1 group_2])
-    end
-
-    it "does not sort the arrays" do
-      expect(result[:group_1].map { |e| e[:base_path] }).to eq(links[:group_1].map { |e| e[:base_path] })
-    end
```

Comment:
suggestion: keep an equivalent of this test in spec/presenters/content_item_presenter_spec.rb to ensure that the `array: false` behaviour.